PICO_STDIO_ENABLE_CRLF_SUPPORT=0 causes a warning with -Wunused

When building a pico-sdk project with `-Wunused` enabled and `PICO_STDIO_ENABLE_CRLF_SUPPORT` set to 0, the following warnings show up: (in my case they are errors due to `-Werror`)
```
.../pico-sdk/src/rp2_common/pico_stdio/stdio.c: In function 'stdio_set_translate_crlf':
.../pico-sdk/src/rp2_common/pico_stdio/stdio.c:306:47: error: unused parameter 'driver' [-Werror=unused-parameter]
  306 | void stdio_set_translate_crlf(stdio_driver_t *driver, bool enabled) {
      |                               ~~~~~~~~~~~~~~~~^~~~~~
.../pico-sdk/src/rp2_common/pico_stdio/stdio.c:306:60: error: unused parameter 'enabled' [-Werror=unused-parameter]
  306 | void stdio_set_translate_crlf(stdio_driver_t *driver, bool enabled) {
      |                                                            ^
```
These do not show up with `PICO_STDIO_ENABLE_CRLF_SUPPORT` set to 1 and `-Wunused` is one of the warnings set in `tests/kitchen_sink`. It seems like this warning should be suppressed by the sdk.

--- a/src/rp2_common/pico_stdio/stdio.c
+++ b/src/rp2_common/pico_stdio/stdio.c
@@ -310,6 +310,10 @@ void stdio_set_translate_crlf(stdio_driver_t *driver, bool enabled) {
     }
     driver->crlf_enabled = enabled;
 #else
+    // Suppress -Wunused-parameter
+    (void)driver;
+    (void)enabled;
+    
     panic_unsupported();
 #endif
 }

